docs/docs/upgrade-gatsby-and-dependencies.md

```diff
@@ -99,21 +97,21 @@ Then add the corresponding script to your package.json file:
 }
 ```
 
-And finally run the recently added command:
+Finalmente, ejecuta el comando que agregaste recientemente en los scripts:
 
 ```shell
 npm run upgrade-interactive
 ```
 
-## Troubleshooting
+## Solución de problemas
 
-Aside from some specific cases, such as [Gatsby's dropping of support for Node 6](/blog/2019-06-18-dropping-support-for-node-6/), upgrading for minor or patch releases should not require you to make changes to your code. It is recommended to run your suite of tests (in case you have one) after upgrading Gatsby or its dependencies.
+Aparte de casos muy específicos como [Gatsby abandonando el soporte para Node 6](/blog/2019-06-18-dropping-support-for-node-6/), actualizar para lanzamientos parche o menores no debería requerir modificaciones a tu código. Se recomienda que corras tu suite de pruebas (en caso de que tengas una) luego de actualizar Gatsby o sus dependencias.
```

Comment:
parte de casos muy específicos como [Gatsby abandonando el soporte para Node 6](/blog/2019-06-18-dropping-support-for-node-6/), actualizar para lanzamientos parche o menores no debería requerir modificaciones a tu código. Se recomienda que corras tu suite de pruebas (en caso de que tengas una), luego de actualizar Gatsby o sus dependencias.